### PR TITLE
Fixed man pages missing install rule

### LINKED ISSUE #19002: man pages are not installed
There are still remnants of it:

https://github.com/OpenRA/OpenRA/blob/71b13c7b5de1ff9682124f92d5345d8a0c67231e/Makefile#L46

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/19002

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -155,6 +158,10 @@ install-linux-shortcuts:
 install-linux-appdata:
 	@sh -c '. ./packaging/functions.sh; install_linux_appdata $(CWD) "$(DESTDIR)" "$(datadir)" cnc d2k ra'
 
+install-man: all
```
Comment (pchote) [inline Makefile:161]: `install-server-manpage` would be clearer.

**Review by atlimit8 — APPROVED** [review]
It works.

**Review by PunkPun — APPROVED** [review]
Works for mac

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b9a1a36e0c15f8b48b32e7bb9676f6f656983db5...e4543de039e92fb9c9f0567b2d9116cd699cbda6)

**Pull request merged**